### PR TITLE
Add JupyterLab Documentation

### PR DESCRIPTION
**Pull request by shreyasbapat**
Addresses #337

### CODE REVIEW

**Comment by shreyasbapat** [comment]
Does plotly version need an update?

**Comment by Juanlu001** [comment]
You tell me! :)

**Comment by Juanlu001** [comment]
Let me rephrase the question: with these instructions but an old release, the plots are still not shown? Can you please check?

**Comment by shreyasbapat** [comment]
The extensions are meant to be for plotly > 3.6 SO yes, I am updating.
